### PR TITLE
Cframe debug

### PR DESCRIPTION
**Pull request by fredericpoitevin**
added an exception catch to deal with gu.CFRAME_PSANA AttributeError

### CODE REVIEW

File: psocake/ImagePanel.py [hunk 1]
```diff
@@ -305,8 +305,8 @@ def updatePolarizationFactor(self): # FIXME: change to vertical polarization aft
 
     def updateDetectorCentre(self):
         self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
-                                                       xy0_off_pix=None,
-                                                       cframe=gu.CFRAME_PSANA, fract=True)
+                                                       xy0_off_pix=None, fract=True)
```
Comment (chuckie82) [inline psocake/ImagePanel.py]: should this be try/except consistent with other changes in this PR?

Comment (fredericpoitevin) [reply psocake/ImagePanel.py]: yes I realized after the fact, it should be fine now

File: psocake/ImagePanel.py [hunk 2]
```diff
@@ -304,9 +307,12 @@ def updatePolarizationFactor(self): # FIXME: change to vertical polarization aft
             if self.parent.args.v >= 1: print "Done updatePolarizationFactor"
 
     def updateDetectorCentre(self):
-        self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
-                                                       xy0_off_pix=None,
+        try:
+            self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
+                                                       xy0_off_pix=None, fract=True,
```
Comment (fredericpoitevin) [inline psocake/ImagePanel.py:312]: Hi @chuckie82 I just realized that one of the `fract=True` arguments need to be removed here.

```bash
Traceback (most recent call last):
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/app/psocake", line 3, in <module>
    from psocake import gui
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/psocake/gui.py", line 16, in <module>
    import MousePanel, ImagePanel, ImageStackPanel
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/psocake/ImagePanel.py", line 313
    cframe=gu.CFRAME_PSANA, fract=True)
SyntaxError: keyword argument repeated
```

Comment (chuckie82) [reply psocake/ImagePanel.py:312]: thanks. if I'm not mistaken I've already fixed this in the latest psocake

Comment (fredericpoitevin) [reply psocake/ImagePanel.py:312]: oh my bad! I checked and indeed you did (I thought I pulled it but did not)

Thanks!

**Pull request merged**